app/views/content_items/topic.html.erb

```diff
@@ -4,82 +4,73 @@
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-      <div<% if @content_item.visually_collapsed? %> class="js-hidden" data-module="accordion-with-descriptions"<% end %>>
-        <div class="subsection-wrapper">
-          <% @content_item.groups.each_with_index do |link_group, idx| %>
-            <div class="subsection js-subsection">
-            <% if link_group.name.present? %>
-              <div class="subsection__header">
-                <h2 class="subsection__title" id="<%= link_group.name.parameterize %>"><%= link_group.name %></h2>
-              <% if link_group.description.present? %>
-                <p class="subsection__description"><%= link_group.description %></p>
-              <% end %>
-              </div>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <% if @content_item.display_as_accordion? %>
+        <%= render "govuk_publishing_components/components/accordion", {
+          items: @content_item.accordion_content
+        } %>
+      <% else %>
+        <% @content_item.groups.each_with_index { |link_group| %>
+          <% if link_group.name.present? %>
+            <div class="subsection__header govuk-!-margin-bottom-3">
+              <h2 class="govuk-heading-m govuk-!-margin-bottom-1 govuk-!-margin-top-3 subsection__title" id="<%= link_group.name.parameterize %>"><%= link_group.name %></h2>
+            <% if link_group.description.present? %>
+              <p class="govuk-body govuk-!-margin-bottom-1 subsection__description"><%= link_group.description %></p>
             <% end %>
-              <div class="subsection__content js-subsection-content" id="subsection_content_<%= idx %>">
-                <ul class="subsection__list">
-                  <% link_group.linked_items.each do |linked_item| %>
-                    <li class="subsection__list-item">
-                      <%= link_to linked_item.label, linked_item.href %>
-                    </li>
-                  <% end %>
-                </ul>
-              </div>
             </div>
           <% end %>
-        </div>
-      </div>
-  </div>
-
-  <div class="column-third">
-    <aside class="related">
-    <% if @content_item.content_owners.any? %>
-      <div class="related-item">
-        <h2 class="related-item__title" id="related-communities">
-          <%= topic_related_communities_title(@content_item.content_owners) -%>
-        </h2>
-        <p class="related-item__description">
-          Find out what the cross-government community does and how to get involved.
-        </p>
-        <nav role="navigation" aria-labelledby="related-communities" class="related-communities">
-          <ul class="related-item__list">
-            <% @content_item.content_owners.each do |content_owner| %>
-              <li class="related-item__list-item">
-                <%= link_to content_owner.title, content_owner.href %>
+          <ul class="govuk-list subsection__list govuk-!-margin-bottom-9">
```

Comment:
looks like subsection list class was used to add more padding and remove text-underline. as we've removed that styling we could remove this class.

Reply:
👍